### PR TITLE
 plan: change the value of `descScanFactor`

### PR DESCRIPTION
**Pull request by winoros**
## What have you changed? (mandatory)

change the value of `descScanFactor` to make it compatible with tikv due to https://github.com/pingcap/tikv/pull/3124 .

Now, if each row has only one version, the reverse scan's performance is near the performance of the speed of scan.
If there's a lot of version, it will become about 1.8x than scan.
So we decrease the factor from `5` to `2`.

## What are the type of the changes (mandatory)?

- Improvement

## How has this PR been tested (mandatory)?

No test.

## Refer to a related PR or issue link (optional)
https://github.com/pingcap/tikv/pull/3124

## Benchmark result if necessary (optional)

See the related pr for details.

### CODE REVIEW

File: plan/find_best_task.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const (
 	netWorkFactor      = 1.5
 	netWorkStartFactor = 20.0
 	scanFactor         = 2.0
-	descScanFactor     = 5 * scanFactor
+	descScanFactor     = 2 * scanFactor
```
Comment (shenli) [inline plan/find_best_task.go]: Please add comments here to explain why desc-scan is two times larger than scan?

**Comment by zz-jason** [comment]
LGTM

**Comment by zz-jason** [comment]
@XuHuaiyu @lamxTyler PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**